concent_api/common/logging.py

```diff
@@ -258,6 +261,47 @@ def log(
         raise TypeError('Unexpected logging level')
 
 
+def log_received_force_payment(
+    logger: Logger,
+    force_payment_message: ForcePayment,
+):
+    log(logger, f"ForcePayment received with {len(force_payment_message.subtask_results_accepted_list)} nested SubtaskResultsAccepted.")
+    for subtask_results_accepted in force_payment_message.subtask_results_accepted_list:
+        log(logger, f"ForcePayment message contains SubtaskResultsAccepted with SUBTASK_ID: {subtask_results_accepted.subtask_id}.")
```

Comment:
please pass `subtask_id` as `log` parameter. This function unifies style of all logs and makes easier to search messages. Please change to:
```python
log(logger, f"ForcePayment message contains SubtaskResultsAccepted", subtask_id=subtask_results_accepted.subtask_id)
```

Reply:
Already outdated